util/find-doc-nits

```diff
@@ -115,7 +115,7 @@ sub name_synopsis()
         } elsif ( $line =~ /enum (\S*) \{/ ) {
             # an enumeration: enum ... {
             $sym = $1;
-        } elsif ( $line =~ /#define ([A-Za-z0-9_]+)/ ) {
+        } elsif ( $line =~ /#(?:define|undef) ([A-Za-z0-9_]+)/ ) {
```

Comment:
Why is this needed?

Reply:
Because `doc/man3/OPENSSL_version_text.pod` has a `#undef` line in the synopsis, and there's no reason why that can't there.